pkg/service/backup_scheduler.go

```diff
@@ -229,7 +235,8 @@ func (h *BackupHandler) runIncrementalBackup(now time.Time) {
 		opts.ModAfter = &lastIncrRunEpoch
 		started := time.Now()
 		slog.Debug("Starting incremental backup", "name", h.routineName)
-		stats = backupService.BackupRun(h.backupRoutine, h.backupIncrPolicy, h.cluster, h.storage, opts)
+		stats = backupService.BackupRun(h.backupRoutine, h.backupIncrPolicy, h.cluster,
```

Comment:
we have so many params here, maybe just pass the whole 'h', WDYT?

Reply:
Right, I've seen that amount of parameters growing. But passing in a BackupHandler doesn't feel the right thing to do. Let's leave this for another refactoring cycle.